File: SocceropenApi/app/src/main/java/com/oscarmena/socceropenapi/SoccerApiService.java
package com.oscarmena.socceropenapi;


import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface SoccerApiService {
    @GET("venues")
    Call<VenuesResponse> getVenues(@Query("api_token") String apiToken);

    @GET("venues")
    Call<VenuesResponse> searchVenues(
            @Query("api_token") String apiToken,
            @Query("name") String venueName
    );
}

File: SocceropenApikotlin/app/src/main/java/com/oscarmena/socceropenapikotlin/SoccerApiService.kt
package com.oscarmena.socceropenapikotlin

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface SoccerApiService {
    @GET("venues")
    fun getVenues(@Query("api_token") apiToken: String): Call<VenuesResponse>

    @GET("venues")
    fun searchVenues(
        @Query("api_token") apiToken: String,
        @Query("name") venueName: String
    ): Call<VenuesResponse>
}
